gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,4 +1,4 @@
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.6-all.zip
```

Comment:
Gradle bug caused build issues. Recommendation online for this bug was upgrade to latest 7.6.x